server/src/main/java/info/doseamigos/meds/MedWebService.java

```diff
@@ -67,6 +67,45 @@ public void createOrUpdateNewMed(
     }
 
     /**
+     * GET call on meds, gets list of meds for specific auth user.
+     * @param inputStream
+     * @param outputStream
+     * @param context
+     * @throws IOException
+     */
+    public void getMedsForSessionUser(
+        InputStream inputStream,
+        OutputStream outputStream,
+        Context context
+    ) throws IOException {
+
+        LambdaLogger logger = context.getLogger();
+
+        //Logging purposes
+        String streamContents = IOUtils.toString(inputStream);
+        logger.log("Stream Contents: " + streamContents);
+
+
+        ClientRequestObject<Object> clientRequestObject = objectMapper.readValue(
```

Comment:
There is no body in the get call here, so I just called it "Object" so that it rendered as whatever.